tests/continental_extension.prm

```diff
@@ -0,0 +1,246 @@
+#### 
```

Comment:
You could remove a good chunk of the parameter file by using:
`include $ASPECT_SOURCE_DIR/cookbooks/continental_extension.prm`